packages/web-components/src/components/notice-choice/notice-choice.ts

```diff
@@ -420,24 +427,32 @@ class NoticeChoice extends StableSelectorMixin(LitElement) {
         const legalTextName = key.replace(/([A-Z]+)/g, '-$1').toLowerCase();
         const mandatoryCheckboxTemplate = html`
           <span>
-            <div class="${prefix}--form-item bx--checkbox-wrapper">
+            <div
+              class="${prefix}--form-item bx--checkbox-wrapper"
+              part="wrapper">
```

Comment:
It took me a minute to see where this was used. In Storybook, if you choose China or Korea for Country, this will show up. It's unique, so I would suggest we add a modifier part to all of these. Also, wrapper is pretty generic, we don't mean a wrapper around the whole component, just the checkbox, so I think it's more appropriate to use the full `checkbox-wrapper` name:
```suggestion
              part="checkbox-wrapper checkbox-wrapper--mandatory">
```